crypto/vrf.go

```diff
@@ -42,6 +42,11 @@ func init() {
 // VRFVerifier is a deprecated name for VrfPubkey
 type VRFVerifier = VrfPubkey
 
+// VRFVerifierMaxSize forwards to base implementation since it's expected by the msgp generated MaxSize functions
+func VRFVerifierMaxSize() int {
+	return VrfPubkeyMaxSize()
+}
```

Comment:
Why is this needed? The bound can be set like elsewhere in the struct field where this is used.

Reply:
because of type alias. This type is on TODO for removal.